streaming/src/data_reader.h

```diff
@@ -11,23 +11,33 @@
 #include "message/message_bundle.h"
 #include "message/priority_queue.h"
 #include "runtime_context.h"
+#include "reliability_helper.h"
 
 namespace ray {
 namespace streaming {
 
-/// Databundle is super-bundle that contains channel information (upstream
-/// channel id & bundle meta data) and raw buffer pointer.
-struct DataBundle {
-  uint8_t *data = nullptr;
-  uint32_t data_size;
-  ObjectID from;
-  uint64_t seq_id;
-  StreamingMessageBundleMetaPtr meta;
+class ReliabilityHelper;
+class AtLeastOnceHelper;
+
+enum class BundleCheckStatus : uint32_t {
```

Comment:
Perphas it's good solution to move all status definition to status.h rather than keep them in different source files.